worker/introspection/socket_test.go

```diff
@@ -107,7 +107,7 @@ func (s *introspectionSuite) TestCmdLine(c *gc.C) {
 }
 
 func (s *introspectionSuite) TestGoroutineProfile(c *gc.C) {
-	buf := s.call(c, "/debug/pprof/goroutine")
+	buf := s.call(c, "/debug/pprof/goroutine?debug=1")
```

Comment:
was this for debugging purposes, what happens on a go 1.6 version if we pass these? Given that the URL is 'debug' it feels weird to have to specify debug=1 again.
And are old clients that are poking this URL (the juju shell scripts) going to be passing the right thing?